tox.ini

```diff
@@ -3,8 +3,9 @@ skipsdist = True
 usedevelop = True
 
 envlist =
-    python{3.8,3.9,3.10}-django{3.2,4.1}-wagtail{4.1,4.2,5.0,5.1,main}-{sqlite,postgres}
-    python{3.11}-django{4.1,4.2}-wagtail{5.0,5.1,main}-{sqlite,postgres}
+    python{3.8,3.9,3.10}-django{3.2,4.1,4.2}-wagtail{4.1,5.1,5.2}-{sqlite,postgres}
```

Comment:
I think here the matrix needs to be broken into 2 sections as Wagtail 4.1 doesn't work with django 4.2. wagtail 5.0 is the first version that supported Django 4.2
It looks like why the tests are failing.

Reply:
Also the merge to main for the 5.1 upgrade didn't run OK.
I think this might have been because of the djangomain issue and we should fix up main first then rebase 5.2 to pick up stopping djangomain for now.

Reply:
Hi @nickmoreton ,
This is a fix PR that should be pushed to the `main` branch. Could you please approve this one?
https://github.com/torchbox-forks/wagtail-rangefilter/pull/5

This PR builds on that.

Thanks!